toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -331,6 +331,7 @@ chroot_and_install_rpms azurelinux-check-macros
 build_rpm_in_chroot_no_install python-rpm-generators
 chroot_and_install_rpms python-rpm-generators python3-rpm-generators
 build_rpm_in_chroot_no_install filesystem
+build_rpm_in_chroot_no_install setup
```

Comment:
It looks like the only non-test BuildRequires for `setup` is `systemd-bootstrap-rpm-macros`. So, make sure to build it after  the line containing `chroot_and_install_rpms systemd-bootstrap`

Reply:
> It looks like the only non-test BuildRequires for `setup` is `systemd-bootstrap-rpm-macros`. So, make sure to build it after the line containing `chroot_and_install_rpms systemd-bootstrap`

okey dokey